src/components/doctors/doctors-list.js

```diff
@@ -1,12 +1,145 @@
-import React from 'react';
-import Navigation from '../nav/navigation';
+import { React, useState } from 'react';
 import './doctors.css';
+import './doctor.css';
+/* eslint-disable */
 
 function DoctorsList() {
+  const [doctors, setdoctors] = useState([
+    {
```

Comment:
- [ ] The Dummy data that needs to be replaced with data coming from the endpoint